addons/account/__init__.py

```diff
@@ -40,7 +40,7 @@ def _auto_install_l10n(env):
             #countries using OHADA Chart of Accounts
             module_list.append('l10n_syscohada')
         elif country_code == 'GB':
-            module_list.append('l10n_uk')
+            module_list.append('l10n_uk', 'account_bacs')
```

Comment:
```suggestion
            module_list.append('l10n_uk')
            module_list.append('account_bacs')
```

Reply:
I will check about the `auto_install`.
But the runbot error says `extend` and not `append`. Please do that instead then.

Reply:
The argument to `extend` has to be a single value: `module_list.extend(('l10n_uk', 'account_bacs'))`

Reply:
For the record: `auto_install` does not work because any module there has to be in the `depends` too